src/test/java/uk/gov/companieshouse/certificates/orders/api/controller/CertificateItemsControllerIntegrationTest.java

```diff
@@ -1798,6 +1826,242 @@ void updateCertificateItemRejectsConflictingDetailsSettings() throws Exception {
+                        .header(ERIC_AUTHORISED_USER_HEADER_NAME, ERIC_AUTHORISED_USER_VALUE)
+                        .header(ERIC_AUTHORISED_TOKEN_PERMISSIONS_HEADER_NAME, String.format(TOKEN_PERMISSION_VALUE, "create"))
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content(objectMapper.writeValueAsString(certificateItemDto)))
+                .andExpect(status().isBadRequest())
+                .andExpect(content().json(objectMapper.writeValueAsString(expectedValidationError)))
+                .andDo(MockMvcResultHandlers.print());
+
+        // Then
+        assertItemWasNotSaved(EXPECTED_ITEM_ID);
+    }
+
+    @Test
+    void correctlyErrorsWhenLiquidatedLimitedCompanyAndGoodStandingInformationSupplied()
+            throws Exception {
+
+        // Given
+        final CertificateItemDTO certificateItemDto = new CertificateItemDTO();
+        certificateItemDto.setCompanyNumber(COMPANY_NUMBER);
+        final CertificateItemOptions options = new CertificateItemOptions();
+        options.setCertificateType(CertificateType.INCORPORATION);
+        options.setIncludeGoodStandingInformation(true);
+        options.setCompanyType("limited");
+        certificateItemDto.setItemOptions(options);
+        certificateItemDto.setQuantity(QUANTITY);
+        when(companyProfileResource.getCompanyStatus()).thenReturn(CompanyStatus.LIQUIDATION);
+        when(companyService.getCompanyProfile(any())).thenReturn(companyProfileResource);
+
+        final ApiError expectedValidationError =
+                new ApiError(BAD_REQUEST,
+                        asList("include_good_standing_information: must not exist when "
+                                + "company status is liquidation"));
+
+        // When and Then
+        mockMvc.perform(post(CERTIFICATES_URL)
+                        .header(REQUEST_ID_HEADER_NAME, TOKEN_REQUEST_ID_VALUE)
+                        .header(REQUEST_ID_HEADER_NAME, TOKEN_REQUEST_ID_VALUE)
+                        .header(ERIC_IDENTITY_TYPE_HEADER_NAME, ERIC_IDENTITY_TYPE_OAUTH2_VALUE)
+                        .header(ERIC_IDENTITY_HEADER_NAME, ERIC_IDENTITY_VALUE)
+                        .header(ERIC_AUTHORISED_USER_HEADER_NAME, ERIC_AUTHORISED_USER_VALUE)
+                        .header(ERIC_AUTHORISED_TOKEN_PERMISSIONS_HEADER_NAME, String.format(TOKEN_PERMISSION_VALUE, "create"))
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content(objectMapper.writeValueAsString(certificateItemDto)))
+                .andExpect(status().isBadRequest())
+                .andExpect(content().json(objectMapper.writeValueAsString(expectedValidationError)))
+                .andDo(MockMvcResultHandlers.print());
+
+        // Then
+        assertItemWasNotSaved(EXPECTED_ITEM_ID);
+    }
+
+    @Test
+    void correctlyErrorsWhenActiveLLPCompanyAndLiquidatorsDetailsSupplied()
```

Comment:
Parameterise test usingParameterizedTest annotation.

Reply:
All six liquidators details tests in this suite are now performed by a single test method. As advised parameterisation has been implemented using the PararameterizedTest annotation. Test parameters are now provided to the test method by a new CertificateItemsFixture [builder] class.